src/commands/app/install/joomla.tsx

```diff
@@ -1,111 +1,38 @@
-import { normalizeToAppVersionUuid } from "../../../lib/app/versions.js";
-import {
-  autofillFlags,
-  provideSupportedFlags,
-  RelevantFlagInput,
-} from "../../../lib/app/flags.js";
-import { waitUntilAppIsInstalled } from "../../../lib/app/wait.js";
-import { MittwaldAPIV2 } from "@mittwald/api-client";
-import { withProjectId } from "../../../lib/project/flags.js";
 import { ExecRenderBaseCommand } from "../../../rendering/react/ExecRenderBaseCommand.js";
-import { makeProcessRenderer } from "../../../rendering/process/process_flags.js";
-import { Success } from "../../../rendering/react/components/Success.js";
 import React from "react";
-import AppAppVersion = MittwaldAPIV2.Components.Schemas.AppAppVersion;
-import { triggerAppInstallation } from "../../../lib/app/install.js";
-import { OutputFlags } from "@oclif/core/lib/interfaces/parser.js";
-
-export default class AppInstallation extends ExecRenderBaseCommand<
-  typeof AppInstallation,
-  { appInstallationId: string }
-> {
-  static appName = "Joomla!";
-  static appUuid = "8d404bff-6d75-4833-9eed-1b83b0552585";
-  static appSupportedFlags = [
+import {
+  AppInstallationResult,
+  AppInstaller,
+} from "../../../lib/app/Installer.js";
+
+const installer = new AppInstaller(
+  "8d404bff-6d75-4833-9eed-1b83b0552585",
+  "Joomla!",
+  [
     "version",
     "host",
-    "admin-firstname",
     "admin-user",
     "admin-email",
     "admin-pass",
     "admin-firstname",
     "admin-lastname",
     "site-title",
     "wait",
-  ] as const;
+  ] as const,
+);
 
-  static description = `Creates new ${AppInstallation.appName} Installation.`;
-  static flags: RelevantFlagInput<typeof AppInstallation.appSupportedFlags> =
-    provideSupportedFlags(
-      AppInstallation.appSupportedFlags,
-      AppInstallation.appName,
-    );
+export default class InstallContao extends ExecRenderBaseCommand<
```

Comment:
This seems to be a consistent copy&paste issue

```suggestion
export default class InstallJoomla extends ExecRenderBaseCommand<
```

Reply:
I completely disregarded this necessity.. its fixed now.